recipe/meta.yaml

```
@@ -42,7 +44,7 @@ about:
     By wrapping node-http-proxy, configurable-http-proxy extends this functionality to 
     JupyterHub deployments.
   dev_url: https://github.com/jupyterhub/configurable-http-proxy
-  doc_url: https://github.com/jupyterhub/configurable-http-proxy
+  doc_url: https://github.com/jupyterhub/configurable-http-proxy/tree/main/doc
```

```suggestion
  doc_url: https://github.com/jupyterhub/configurable-http-proxy/tree/main/README.md
```

There's no actual documentation in /doc.